model/src/main/java/org/alfresco/transform/common/RequestParamMap.java

```diff
@@ -1,93 +1,96 @@
+{
+    // html parameter names
+    String FILE = "file";
+    String SOURCE_EXTENSION = "sourceExtension";
+    String TARGET_EXTENSION = "targetExtension";
+    String SOURCE_MIMETYPE = "sourceMimetype";
+    String TARGET_MIMETYPE = "targetMimetype";
+
+    // Transform options used in the core transformers.
+    String SOURCE_ENCODING = "sourceEncoding";
+    String TARGET_ENCODING = "targetEncoding";
+    String PAGE_REQUEST_PARAM = "page";
+    String WIDTH_REQUEST_PARAM = "width";
+    String HEIGHT_REQUEST_PARAM = "height";
+    String ALLOW_PDF_ENLARGEMENT = "allowPdfEnlargement";
+    String MAINTAIN_PDF_ASPECT_RATIO = "maintainPdfAspectRatio";
+    String START_PAGE = "startPage";
+    String END_PAGE = "endPage";
+    String ALPHA_REMOVE = "alphaRemove";
+    String AUTO_ORIENT = "autoOrient";
+    String CROP_GRAVITY = "cropGravity";
+    String CROP_WIDTH = "cropWidth";
+    String CROP_HEIGHT = "cropHeight";
+    String CROP_PERCENTAGE = "cropPercentage";
+    String CROP_X_OFFSET = "cropXOffset";
+    String CROP_Y_OFFSET = "cropYOffset";
+    String THUMBNAIL = "thumbnail";
+    String RESIZE_WIDTH = "resizeWidth";
+    String RESIZE_HEIGHT = "resizeHeight";
+    String RESIZE_PERCENTAGE = "resizePercentage";
+    String ALLOW_ENLARGEMENT = "allowEnlargement";
+    String MAINTAIN_ASPECT_RATIO = "maintainAspectRatio";
+    String COMMAND_OPTIONS = "commandOptions";
+    String TIMEOUT = "timeout";
+    String INCLUDE_CONTENTS = "includeContents";
+    String NOT_EXTRACT_BOOKMARKS_TEXT = "notExtractBookmarksText";
+    String PAGE_LIMIT = "pageLimit";
+    String PDF_FORMAT = "pdfFormat";
+    String PDF_ORIENTATION = "pdfOrientation";
+    String PDF_FONT = "pdfFont";
+    String PDF_FONT_SIZE = "pdfFontSize";
+
+    // Parameters interpreted by the TransformController
+    String DIRECT_ACCESS_URL = "directAccessUrl";
+
+    // An optional parameter (defaults to 1) to be included in the request to the t-engine {@code /transform/config}
+    // endpoint to specify what version (of the schema) to return. Provides the flexibility to introduce changes
+    // without getting deserialization issues when we have components at different versions.
+    String CONFIG_VERSION = "configVersion";
+    String CONFIG_VERSION_DEFAULT = "1";
+    int CONFIG_VERSION_LATEST = CoreVersionDecorator.CONFIG_VERSION_INCLUDES_CORE_VERSION;
+
+    // Endpoints
+    String ENDPOINT_TRANSFORM = "/transform";
+    String ENDPOINT_TEST = "/test";
+    String ENDPOINT_TRANSFORM_CONFIG = "/transform/config";
+    String ENDPOINT_TRANSFORM_CONFIG_LATEST = ENDPOINT_TRANSFORM_CONFIG + "?" + CONFIG_VERSION + "=" + CONFIG_VERSION_LATEST;
+    String ENDPOINT_VERSION = "/version";
+    String ENDPOINT_READY = "/ready";
+    String ENDPOINT_LIVE = "/live";
+    String ENDPOINT_ERROR = "/error";
+    String ENDPOINT_LOG = "/log";
+    String ENDPOINT_ROOT = "/";
+
+    // Html parameter names for the transform config
+    String HTML_COLLAPSE = "collapseHtml";
```

Comment:
Just a minor change for consistency purposes: could this variable be placed below the `PDF_FONT_SIZE` one?